crates/sui-deepbook-indexer/src/sui_deepbook_indexer.rs

```diff
@@ -461,6 +461,35 @@ fn process_sui_event(
                     balance_manager_id: move_event.balance_manager_id.to_string(),
                 }))
             }
+            "OrderExpired" => {
+                info!("Observed Deepbook Order Expired {:?}", ev);
+                // metrics.total_sui_token_deposited.inc();
+                let move_event: MoveOrderExpiredEvent = bcs::from_bytes(&ev.contents)?;
+                let txn_kind = tx.transaction.transaction_data().clone().into_kind();
+                let first_command = txn_kind.iter_commands().next();
+                let package = if let Some(Command::MoveCall(move_call)) = first_command {
+                    move_call.package.to_string()
+                } else {
+                    "".to_string()
+                };
```

Comment:
curious why we only look at the first command?

Reply:
I couldn't figure out how we would identify the relevant package in a PTB so I went with the first one as a quick solution.

Reply:
i doubt the first command is the most most prominent one. for example, sometimes a ptb starts with `make_move_vec`. We can record all packages (excluding 0x2 for example) used in a PTB if that is helpful. This is not blocking though!